.github/workflows/prtitle.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
         - name: Install Dependencies
           run: |
             python -m pip install --upgrade pip
-            pip install .[dev]
+            pip install "commitizen>=2.40,<2.41"
```

Comment:
The action runs much faster this way and saves action minutes.
It only needs the one tiny tool to do this job! It does not need Ape installed.

Reply:
Yes this is true! I wanted it to source the same version of commitizen from setup.py, only reason I changed it that way

Reply:
Should we go back? I am good with other. Pros and cons!

Reply:
so like if it could just read that line directly from `setup.py` that would be best

Reply:
[f35f8eb](https://github.com/ApeWorX/ape/pull/1267/commits/f35f8ebe9d227a906d4b245c151e15d573dcaf3e)